app/src/main/java/uw/hcrlab/kubi/MainActivity.java

```diff
@@ -46,100 +39,34 @@ protected void onCreate(Bundle savedInstanceState) {
         //Notice: this is how each activity will get the robot and connect it to the robot face
         getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
         setContentView(R.layout.activity_main);
-        robot = Robot.getInstance((RobotFace)findViewById(R.id.face), this);
-    }
-
-    /*
```

Comment:
Well, technically, that's not gone. I just moved the robot creation stuff into a Factory to separate initialization logic from robot logic.